Navidux/Sources/Navidux/NavigationController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  NavigationController.swift
+//  NaviduxTests
+//
+//  Created by Александр Евсеев on 12.10.2022.
+//  Copyright © 2022 red_mad_robot. All rights reserved.
+//
```

Comment:
You need to decide do you need default comments for source files or not. Make it consistent😉

Reply:
remove unnecessary comments)